policyengine_us/variables/gov/states/hi/tax/income/credits/food_excise/hi_food_excise_exemption_amount.py

```diff
@@ -12,6 +12,8 @@ def formula(tax_unit, period, parameters):
         p = parameters(period).gov.states.hi.tax.income.credits.food_excise_tax
 
         income = tax_unit("adjusted_gross_income", period)
+
+        minor_child_include = p.minor_child.availability
```

Comment:
make an if-statement using the p.minor_child.availability 

Calculate in the if-the steps pre 2023 and the rest outside